skimage/filters/thresholding.py

```diff
@@ -1282,15 +1282,17 @@ def threshold_multiotsu(image=None, classes=3, nbins=256, *, hist=None):
     # calculating the histogram and the probability of each gray level.
     prob, bin_centers = _validate_image_histogram(image, hist, nbins,
                                                   normalize=True)
-    prob = prob.astype('float32')
+    prob = prob.astype('float32', copy=False)
 
     nvalues = np.count_nonzero(prob)
     if nvalues < classes:
-        msg = (f'The input image has only {nvalues} different values. '
-               f'It cannot be thresholded in {classes} classes.')
+        msg = (f'After discretization into bins, the input image has '
+               f'only {nvalues} different values. It cannot be thresholded '
+               f'in {classes} classes. If there are more unique values '
+               f'before discretization, try increasing the number of bins.')
```

Comment:
```suggestion
               f'before discretization, try increasing the number of bins (`nbins`).')
```
How does that sound?

Reply:
Thank you @alexdesiqueira :wink: